.gitignore

```diff
@@ -25,3 +25,6 @@ build/Release
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+
+# Ignore package-lock for now. Perhaps at a later date we'll allow this.
```

Comment:
Why are we ignoring this? I find it quite useful

Reply:
I agree. It's very useful. I was going to chat with you about greenkeeper.io in a future issue and I find that keeping package-lock in-sync when using greenkeeper is trickier than vanilla use of greenkeeper. I was thinking that we would start out with the easy use case for greenkeeper and then migrate onto the scenario where we give greenkeeper write access to these repos so that it can update the package-lock file. Hence my reason for wanting to keep it out of the repo until that scenario is complete. WDYT?

Reply:
I'd leave it in for now it's useful for travis. I've never used greenkeeper but it does look useful, though I'd want to review it more thoroughly before giving it write access to the repos. The passport modules don't have many deps so manually updating them isn't a massive challenge right now, especially when we're still forking some.

Best to create a new issue for greenkeeper remove the gitignore line.

Reply:
@rwky - sounds good - done!